data/headers/windows/base64.h

```diff
@@ -17,7 +17,7 @@ static unsigned char alphabet[64] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopq
 int base64decode(char *dest, const char *src, int l)
 {
   static char inalphabet[256], decoder[256];
-  static bool table_initialized = false;
+  static short table_initialized = 0;
```

Comment:
Was changing the type here necessary to compile with MinGW or something else?

Reply:
Yes since ANSI-C does not contain boolean type, it was either including stdbool or changing it to this